Marlin/example_configurations/Hephestos_2/Configuration_adv.h

```diff
@@ -352,6 +352,60 @@
   #define ULTIPANEL_FEEDMULTIPLY  // Comment to disable setting feedrate multiplier via encoder
 #endif
 
+//
+// Custom BQ Bootsplash
+//
+#define SHOW_CUSTOM_BOOTSCREEN
+
+#if ENABLED(SHOW_BOOTSCREEN) && ENABLED(SHOW_CUSTOM_BOOTSCREEN)
+  #include <avr/pgmspace.h>
+
+  #define CUSTOM_BOOTSCREEN_TIMEOUT   2500
+  #define CUSTOM_BOOTSCREEN_BMPWIDTH  37
+  #define CUSTOM_BOOTSCREEN_BMPHEIGHT 38
+
+  const unsigned char custom_start_bmp[190] PROGMEM = {
+    0x00, 0x00, 0x30, 0x00, 0x00,
+    0x00, 0x00, 0x78, 0x00, 0x00,
+    0x00, 0x00, 0xfc, 0x00, 0x00,
+    0x00, 0x00, 0xfc, 0x00, 0x00,
+    0x00, 0x60, 0x78, 0x18, 0x00,
+    0x00, 0xf0, 0x30, 0x3c, 0x00,
+    0x01, 0xf8, 0x00, 0x7e, 0x00,
+    0x01, 0xf8, 0x00, 0x7e, 0x00,
+    0x00, 0xf0, 0x00, 0x3c, 0x00,
+    0x00, 0x60, 0x00, 0x18, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x60,
+    0x00, 0x00, 0x00, 0x00, 0xf0,
+    0x00, 0x00, 0x00, 0x01, 0xf8,
+    0x00, 0x00, 0x00, 0x01, 0xf8,
+    0x00, 0x00, 0x00, 0x00, 0xf0,
+    0x00, 0x00, 0x00, 0x00, 0x60,
+    0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00,
+    0x30, 0x00, 0x00, 0x00, 0x00,
+    0x78, 0x00, 0x18, 0x00, 0x00,
+    0xfc, 0x00, 0x18, 0x00, 0x00,
+    0xfc, 0x00, 0x18, 0x00, 0x00,
+    0x78, 0x00, 0x1f, 0xc3, 0xc0,
+    0x30, 0x00, 0x1c, 0xe6, 0x60,
+    0x00, 0x00, 0x18, 0x6c, 0x30,
+    0x00, 0x00, 0x18, 0x2c, 0x30,
+    0x00, 0x00, 0x18, 0x2c, 0x30,
+    0x00, 0x00, 0x18, 0x6c, 0x30,
+    0x00, 0x00, 0x1c, 0x66, 0x70,
+    0x00, 0x00, 0x07, 0xc7, 0xf0,
+    0x00, 0x00, 0x00, 0x00, 0x30,
+    0x00, 0x00, 0x00, 0x00, 0x30,
+    0x00, 0x00, 0x00, 0x00, 0x30,
+    0x00, 0x00, 0x00, 0x00, 0x30
+  };
+#endif
+
```

Comment:
Could go both ways, but the idea was really to simplify.
This means IMO not to add additional files, thus reusing exactly the same concept we have for Deltas. Their config files are the only ones having some sections which are dedicated to them.

As this feature targets vendors we won't be seeing much of it on the wild I guess.